src/mscorlib/src/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -82,7 +82,7 @@ private unsafe int GetHashCodeOfStringCore(string source, CompareOptions options
 
             fixed (char* pSource = source)
```

Comment:
The randomized string hashing should be unconditionally enabled in .NET Core. It should not matter what you pass in here, it will overridden by `s_NlsHashProvider.GetUseRandomHashing()` in the QCall.